### DISCUSSION TITLE
How to use Allure js mocha with Cypress ?

### DISCUSSION THREAD

**Discussion by skirankumars31** [allure-framework/allure2#2357] [Questions & Support] [answered]
I heard that Mocha reporters can be used directly with Cypress because Cypress is based on Mocha. How do i use the Allure js directly with Cypress ?

**Comment by epszaw** [comment, marked as answer]
Hey, @skirankumars31! You can use our official [Cypress integration](https://github.com/allure-framework/allure-js/tree/main/packages/allure-cypress) to get access to Allure runtime API.
